### PR TITLE
QPPA-3573 - PY2020 IA measure updates

### PR DESCRIPTION
**Pull request by chouinar**
https://jira.cms.gov/browse/QPPA-3573

#### What is being changed

Minor description updates

#### Release checklist:
Tasks that must be done prior to merging this PR, including testing.

* [x] [Package version updated](https://github.com/CMSgov/qpp-measures-data/blob/master/CONTRIBUTING.md#versioning-publishing-and-creating-new-releases)
* [x] Documentation updated
* [x] Unit tests added/passing
* [x] Verified working locally

### CODE REVIEW

File: util/measures/2020/ia-measures-old.csv [hunk 1]
```diff
@@ -0,0 +1,230 @@
+title,description,measureId,subcategoryId,weight,metricType,firstPerformanceYear
```
Comment (sheldon-b) [inline util/measures/2020/ia-measures-old.csv]: Is this file needed?

Comment (chouinar) [reply util/measures/2020/ia-measures-old.csv]: Nope, I copy the old one to a temp file so I can reference it for weird character issues, meant to delete it. Fixed.
